src/i2c_arduino_nano.cpp

```diff
@@ -0,0 +1,35 @@
+#include "CompDevSDK.h"
+
+#ifdef PLATFORM_ARDUINO_NANO
+
+#include <Wire.h>
+#include "delay.h"
+#include "i2c.h"
+
+void i2cInit()
+{
+    Wire.begin();
+}
+
+void i2cSend(uint8_t addr, uint8_t data)
```

Comment:
Najgorsza implementacja i2c. Napisz to na przerwaniach tak żeby nie blokowało zasobów i wykorzystało możliwości FreeRTOS, np task stworzony do obsługi i2c